alacritty/src/config/mod.rs

```diff
@@ -204,6 +204,13 @@ fn print_deprecation_warnings(config: &Config) {
              lines"
         );
     }
+
+    if config.scrolling.auto_scroll.is_some() {
+        warn!(
+            target: LOG_TARGET_CONFIG,
+            "Config scrolling.auto_scroll has been removed, and is now always disabled"
```

Comment:
I still really liked the instruction that people could safely remove the config option. I think it's worthwhile to put that in, because even if a user has absolutely no clue, that provides a clear instruction for finding a solution.

```suggestion
            "Config scrolling.auto_scroll has been removed and is now always disabled, it can be safely removed from the config"
```